internal/pkg/controllers/profile/profile.go

```diff
@@ -121,12 +121,9 @@ type Reconciler struct {
 }
 
 // Reconcile reconciles a SeccompProfile or a ConfigMap representing a seccomp profile.
-func (r *Reconciler) Reconcile(req reconcile.Request) (reconcile.Result, error) {
+func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reconcile.Result, error) {
 	logger := r.log.WithValues("profile", req.Name, "namespace", req.Namespace)
 
-	ctx, cancel := context.WithTimeout(context.Background(), reconcileTimeout)
```

Comment:
Hmm don't we want this context to be per-reconcile?

Reply:
@saschagrunert we are not actively working on it (updated fairly recently), but we would love for you to contribute it :)